File: 3959-check-good-integer/3959-check-good-integer.py
class Solution:
    def checkGoodInteger(self, n: int) -> bool:
        digitSum=0
        squareSum=0
        while(n>0):
            rem=n%10
            digitSum+=rem
            squareSum+=(rem*rem)
            n//=10
        return True if (squareSum-digitSum)>=50 else False 

File: 3959-check-good-integer/3959-check-good-integer.cpp
class Solution {
public:


    bool checkGoodInteger(int n) {
        int temp=n;
        int digitSum=0;
        int squareSum=0;
        while(n){
            int rem=n%10;
            digitSum+=rem;
            squareSum+=(rem*rem);
            n/=10;
        }
        return (squareSum-digitSum)>=50 ? true :false;
    }
};